<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Injee" failures="3" tests="5" name="Test" time="44.017" errors="0" timestamp="2025-03-13T17:03:32 CDT">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="12.102">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.916">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="3.18">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.687"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="3.268"/>
</testsuite> <!-- Test -->
